packages/perseus/src/widgets/interactive-graphs/graphs/components/angle-indicators.test.ts

```diff
@@ -94,10 +95,13 @@ describe("shouldDrawArcOutside", () => {
         ).toBe(true);
     });
 
+    // TODO: (third) Move this test to the math package
     it("should correctly calculate the angle for the given coordinates", () => {
         const point1 = [2, 2] as vec.Vector2;
         const point2 = [2, 0] as vec.Vector2;
         const vertex = [0, 0] as vec.Vector2;
-        expect(findAngle(point1, point2, vertex)).toBe(45);
+        const coords: [Coord, Coord, Coord] = [point1, vertex, point2];
+        const allowReflexAngles = false;
```

Comment:
Question: Does this still need to be defined after the change Anakaren suggested where the default is false? Haven't used defaults much, but I think it should just be false if the value is undefined, right? I could see it just being more clear to state it outright in a test, but just wanted to check :)

Reply:
No it doesn't, I think you're right!